### PR TITLE
[ig/security] QA and referencing to Groups dealing with Security

### PR DESCRIPTION
**Pull request by simoneonofri**
thx to @msporny

### CODE REVIEW

File: 2024/ig-security.html [hunk 1]
```diff
@@ -151,7 +151,7 @@ <h2>Motivation and Background</h2>
...
+        <p>Several working groups deal with security issues, such as <a href="https://www.w3.org/groups/wg/webappsec/">developing mechanisms and best practices which improve the security of Web Applications</a>, <a href="https://www.w3.org/groups/wg/webauthn/">developing strong authentication functionality for Web Applications</a>, <a href="https://www.w3.org/groups/wg/fedid/">developing APIs to allow a website to request an identity credential securely</a>, and <a href="https://www.w3.org/groups/ig/securepay/">enhancing the security and interoperability of various Web payments technologies</a>.</p>
```
Comment (msporny) [inline 2024/ig-security.html]: If we are going to list application layer things (like payments and fedcm) we should also list the VCWG:

```suggestion
        <p>Several working groups deal with security issues, such as <a href="https://www.w3.org/groups/wg/webappsec/">developing mechanisms and best practices which improve the security of Web Applications</a>, <a href="https://www.w3.org/groups/wg/webauthn/">developing strong authentication functionality for Web Applications</a>, <a href="https://www.w3.org/groups/wg/fedid/">developing APIs to allow a website to request an identity credential securely</a>, <a href="https://www.w3.org/groups/ig/securepay/">enhancing the security and interoperability of various Web payments technologies</a>, and <a href="https://www.w3.org/groups/wg/vc/">creating data models and securing mechanisms for verifiable credentials</a>.</p>
```

**Review by msporny — CHANGES_REQUESTED** [review]
If we are going to list application layer things (like payments and fedcm) we should also list the VCWG.

**Comment by simoneonofri** [comment]
@msporny The list was not meant to be exhaustive but to give more context to the group versus the various security-related activities within W3C, so it's not a problem for me to include the Data Integrity part.

cc'ing @tantek @jyasskin @chrisn

**Review by tantek — CHANGES_REQUESTED** [review]
@simoneonofri this section seems like the wrong place for such a long list. Maybe reduce that list here to only the groups that literally have "security" or Sec in their name/shortname.

Then instead, move all the groups in this list that explicitly "deal with security issues" (e.g. their charters mention how and what they do to deal with security issues) to the Coordination / Liaison section, in individual list-items there (one per group), with a brief phrase / sentence of the intended goal of the coordination with each specific group.

**Comment by simoneonofri** [comment]
@tantek @msporny

I've thought about it a little bit. The background section is probably not critical, but it's there to give context.  Reasoning about the “resilience” of the charter, as with other issues/PRs, there are two options:
 a. Leaving it with the list
 b. Referring directly to the new Security page https://www.w3.org/mission/security/
 
 what do you think?

**Comment by msporny** [comment]
> there are two options: a. Leaving it with the list b. Referring directly to the new Security page https://www.w3.org/mission/security/
> what do you think?

I would be fine with either since they both list the groups involved. 

I will note that the /mission/security/ page does not include the VCWG and it probably should. Once that is done, referring to /missions/security/ is probably best since it can be updated, whereas the charter cannot be easily updated.

/cc @brentzundel @iherman

**Comment by simoneonofri** [comment]
Thank you @msporny 

The page does not list them, but I need to make several updates. I also need to talk with communication.

I like @tantek's “application” security categorization.

**Comment by simoneonofri** [comment]
@tantek @msporny I updated the charter text, referencing the W3C Security Mission page. I'll work in parallel with the Comm Team to update it.

**Pull request merged**